CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## Handle Granules with Identical producerGranuleId in Different Collections
 
+### Breaking Changes
+
+- **CUMULUS-4072**
+  - Updated the `parse-pdr` task component to throw an error if multiple granules within the same PDR have the same granuleId after applying the granuleIdFilter, unless the uniquifyGranules configuration parameter is explicitly set to `true`.
```

Comment:
Which is the configuration parameter uniquifyGranules or uniquifyGranuleId?

Reply:
Good catch: 5dcfa3530cc